pkg/standalone/standalone.go

```diff
@@ -327,13 +346,22 @@ func runRedis(wg *sync.WaitGroup, errorChan chan<- error, info initInfo) {
 	}
 	args := []string{}
 
+	var imageName string
 	if exists {
 		// do not create container again if it exists.
 		args = append(args, "start", redisContainerName)
 	} else {
-		imageName := redisDockerImageName
-		if info.imageRepositoryURL != "" {
-			imageName = fmt.Sprintf("%s/%s", info.imageRepositoryURL, imageName)
+		if defaultImageRegistryName == githubContainerRegistryName {
+			imageName = redisGhcrImageName
+			if info.imageRegistryURL == "" {
+				info.imageRegistryURL = ghcrURL
+			}
+		} else if defaultImageRegistryName == dockerContainerRegistryName {
+			imageName = redisDockerImageName
+		}
```

Comment:
Can we abstract this in a separate function and use this for redis and zipkin.
Adding a unit test on the same might help.

This will be useful in future if we need to run any other component